janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java

```diff
@@ -275,6 +275,8 @@ public boolean apply(Integer input) {
     private final boolean skipSchemaCheck;
     private final String compatClass;
     private final HBaseCompat compat;
+    // Cached return value of getDeployment() as requesting it can be expensive.
```

Comment:
That's something I'm not sure about.  I'm interested to hear from Jerry on this. In the Bigtable case I'm not sure it would ever change, as AFAIK it doesn't expose the key ranges through this method. For Hbase though, does it change?  And how often?